firebase.json

```diff
@@ -24,1610 +24,648 @@
       {
-        "source": "/:lang/docs/guides/analytics/analytics_basics",
-        "destination": "/:lang/docs/analytics/analytics_basics.html",
+        "source": "/:lang/docs/getting_started/create",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/ads_analytics/analytics_basics",
-        "destination": "/:lang/docs/analytics/analytics_basics.html",
+        "source": "/:lang/docs/getting_started/create/basic_markup",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create/basic_markup?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/analytics/analytics-vendors",
-        "destination": "/:lang/docs/analytics/analytics-vendors.html",
+        "source": "/:lang/docs/getting_started/create/include_image",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create/include_image?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/ads_analytics/analytics-vendors",
-        "destination": "/:lang/docs/analytics/analytics-vendors.html",
+        "source": "/:lang/docs/getting_started/create/prepare_for_discovery",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create/prepare_for_discovery?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/ads_analytics/ads-vendors",
-        "destination": "/:lang/docs/ads/ads-vendors.html",
+        "source": "/:lang/docs/getting_started/create/presentation_layout",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create/presentation_layout?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/ads_analytics/integration-guide",
-        "destination": "/:lang/docs/ads/integration-guide.html",
+        "source": "/:lang/docs/getting_started/create/preview_and_validate",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create/preview_and_validate?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/analytics/deep_dive_analytics",
-        "destination": "/:lang/docs/analytics/deep_dive_analytics.html",
+        "source": "/:lang/docs/getting_started/create/publish",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/create/publish?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/ads_analytics/deep_dive_analytics",
-        "destination": "/:lang/docs/analytics/deep_dive_analytics.html",
+        "source": "/:lang/docs/getting_started/quickstart",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/?referrer=ampproject.org",
```

Comment:
feels like there should be a better link for t his

Reply:
What do you think of: https://amp.dev/documentation/guides-and-tutorials/start/create.html?format=websites?

Reply:
sgtm